aws-distro-opentelemetry-node-autoinstrumentation/test/patches/instrumentation-patch.test.ts

```diff
@@ -56,6 +62,10 @@ describe('InstrumentationPatchTest', () => {
     expect(services.has('DynamoDB')).toBeTruthy();
     expect(services.has('Lambda')).toBeTruthy();
     // From patching but shouldn't be applied
```

Comment:
Move SNS and Lambda from
- `// From patching but shouldn't be applied`

into
- `// Not from patching`